daemon/policy.go

```
@@ -116,8 +116,11 @@ func (d *Daemon) traceL4Egress(ctx policy.SearchContext, ports []*models.Port) a
 	ctx.EgressL4Only = true
 
 	ctx.PolicyTrace("\n")
-	policy := d.policy.ResolveL4Policy(&ctx)
-	verdict := policy.EgressCoversDPorts(ports)
+	policy, err := d.policy.ResolveL4Policy(&ctx)
+	verdict := api.Undecided
+	if err == nil {
```

Aren't you ignoring the error, in case there's one?

---

The caller will report 'api.Undecided' return as denied. mergeL4() during the ResolveL4Policy will trace the conflict.